src/types.ts

```diff
@@ -12,6 +12,8 @@ export interface Video {
     onProgress?: (arg0: VideoProgress) => void
     channelName?: string
     uploadAsDraft?: boolean
+    isRestriction?: boolean
```

Comment:
"isVideoAgeRestricted" might be a better name.

I had to look at the code to understand what this property does.

Reply:
i've change to `isNotForKid` and `isAgeRestriction`